Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM kicad/kicad:8.0
+FROM kicad/kicad:9.0.0-amd64
```

Comment:
Please use the multi-platform variant [kicad/kicad:9.0](https://hub.docker.com/layers/kicad/kicad/9.0/images/sha256-f2577a93efda88763a4129d02ec8f750760d5588d4e769803ba542b46a905683), Docker will then figure out the architecture itself.